### PR TITLE
fix: caption hiding behind image issue

### LINKED ISSUE #1281: Caption hidden
### Description

Captions written alongside an uploaded image in portrait hides behind the image. Kindly assign under hacktoberfest label

### Steps to reproduce

Upload a picture in portrait mode, Add a caption, click on post. On the homepage, the image overshadows it

### Screenshots

Screenshot from 2023-10-18 09-39-26

### Desktop (please complete the following information)

ubuntu, Firefox

### Smartphone (please complete the following information)

_No response_

### Additional information

_No response_

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I have checked the existing issues
- [ ] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by 0xNunana**
This fix closes #1281 
Screenshot from 2023-10-20 00-48-59

### CODE REVIEW

File: src/components/ReadMore/index.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ const ReadMore = ({ children, postId, picCap = false, readMore = true }) => {
     : text;
 
   return (
-    <>
+    <div style={{zIndex:1,backgroundColor:'white',opacity:0.7}}>
```
Comment (narayan954) [inline src/components/ReadMore/index.jsx:21]: please avoid inline styling as it reduces readability. You may add a class to this div with suitable name

Comment (narayan954) [reply src/components/ReadMore/index.jsx:21]: You may also want to check out the other occurrences of this component and see if they're mis-affected.

**Review by narayan954 — CHANGES_REQUESTED** [review]
kindly go through the comments. Minor changes requested

File: src/components/ReadMore/index.jsx [hunk 2]
```diff
@@ -18,7 +20,7 @@ const ReadMore = ({ children, postId, picCap = false, readMore = true }) => {
     : text;
 
   return (
-    <>
+    <div className="caption_container">
```
Comment (narayan954) [inline src/components/ReadMore/index.jsx:23]: please check the console, there's an error `<div> cannot appear as a descendant of <p>` perhaps due to this component being rendered inside a paragraph element.

**Review by narayan954 — CHANGES_REQUESTED** [review]
please check where this component is used and fix the error.

**Review by 0xNunana — COMMENTED** [review]
changed the div into a span

**Comment by narayan954** [comment]
Thanks for your contribution! :)

**Pull request merged**